eng/pipelines/common/platform-matrix.yml

```diff
@@ -47,18 +47,18 @@ jobs:
         ${{ if eq(parameters.passPlatforms, true) }}:
           platforms: ${{ parameters.platforms }}
         helixQueueGroup: ${{ parameters.helixQueueGroup }}
-        managedTestBuildOsGroup: Linux
+        managedTestBuildOsGroup: linux
```

Comment:
@Anipik did we miss these when you renamed OSGroup to BuildOS?

Reply:
I didnt change the .yml scripts variable names in the previous PR. i Wanted to keep that little bit separate as other change was getting crowded.